tests/data/ext/sample_ext-1.3.0/sample.py

```diff
@@ -1,37 +1,96 @@
 #!/usr/bin/env python
-
+import json
 import os
+import re
+import sys
+
 
+def get_seq(requested_ext_name=None):
```

Comment:
Modified the `sample_ext` extension to account for MultiConfig. This would now report properly for both Multi and Single-config extensions and would also parse and output the message from the `publicSettings` of the settings file. This ensures that the guest agent is correctly writing the settings file for extensions too